resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -3619,6 +3619,870 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
           <hashTree/>
         </hashTree>
       </hashTree>
+      <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Login (Execution)" enabled="true">
+        <stringProp name="ThreadGroup.on_sample_error">continue</stringProp>
+        <elementProp name="ThreadGroup.main_controller" elementType="LoopController" guiclass="LoopControlPanel" testclass="LoopController" testname="Loop Controller" enabled="true">
+          <boolProp name="LoopController.continue_forever">false</boolProp>
+          <stringProp name="LoopController.loops">1</stringProp>
+        </elementProp>
+        <stringProp name="ThreadGroup.num_threads">1</stringProp>
+        <stringProp name="ThreadGroup.ramp_time">1</stringProp>
+        <boolProp name="ThreadGroup.scheduler">false</boolProp>
+        <stringProp name="ThreadGroup.duration"></stringProp>
+        <stringProp name="ThreadGroup.delay"></stringProp>
+        <boolProp name="ThreadGroup.same_user_on_next_iteration">true</boolProp>
+      </ThreadGroup>
+      <hashTree>
+        <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="Login API" enabled="true">
```

Comment:
Load Coordinates From File - we have so many details but we are using only 4 in the "Registeration Center List API (Nearest Location Coordinates)" - why this this ?

Reply:
if the remaining is for "Registeration Center List API (Location Hierarchy)", then lets maintain in 2 different files, maintaining in one with no relavance is confusing

Reply:
Have created a common file naming as coordinates, where have added many details which are used in other API's. So out of that 4 values have been used here.

There are over all 10 APIs where different data is required from this file(Coordinates). So, rather than using so many different test data files I have created a common file for all those API's.

Reply:
But if the data is not corelated, then the samples taken are confused for someone to follow

Reply:
Its not updated in the latest commit.

Reply:
As discussed, lets split to multiple files

Reply:
Done

Reply:
New files will be used for other API's which are not in this PR. And two variables were having same value which is locationCode and hierarchyLevel so have added these two in the user defined variables.

Reply:
locationCode and hierarchyLevel - these will have multiple values, it cannot be given a single value

Reply:
okay, have created a separate file for hierarchy level.